sample-service/src/main/java/uk/ac/ox/ndph/mts/sampleservice/DemoApp.java

```diff
@@ -1,6 +1,6 @@
-package com.microsoft.baseapp;
+package uk.ac.ox.ndph.mts.sampleservice;
```

Comment:
I believe this should be sample_service. isn't it?

Reply:
@tamirkamara  yes, pls resolve